staff/sys/migrate-vm

```diff
@@ -45,23 +42,25 @@ from ocflib.misc.shell import red
 
 HostAndVm = namedtuple('HostAndVm', ['host', 'vm'])
 
+client = None
+
 
 def exec_command(cmd, ignore_status_code=False):
-    print(bold(cmd))
```

Comment:
So, I do agree, and I was going to add `-v` and `-vv` (or probably `-quiet` and `-debug`) to print state-changing commands and all commands, respectively, as a PR. I inserted the stupid "I'm too cool to do it myself" tone to this comment I guess to save myself more explanation.

The command as it is prints absolutely none of the state-changing commands it performs on either host (except `virsh shutdown`), and no commands at all on the current host, which is pretty silly. It basically just needs a bunch of print statements inserted for the local commands in addition to this one here, but I thought I'd add log levels too since not all the commands called have any visible effect.